credible/faq.mdx

```diff
@@ -105,41 +99,39 @@ beyond standard Solidity.
 For detailed documentation and examples of all available cheatcodes, see our 
 [Cheatcodes Reference Guide](/credible/cheatcodes).
 
-### How do I upgrade assertions?
+### How do I edit assertions?
 
-There might be cases where you want to upgrade an assertion to add new checks or modify existing ones.
+There might be cases where you want to edit an assertion to add new checks or change existing ones.
 
-Upgrading an assertion is straightforward:
-1. Disable current assertion
-2. Deploy new assertion pointing and point it to the contract you're protecting
+Editing an assertion is straightforward:
+1. Disable the current assertion
+2. Deploy new assertion and point it to the contract you're protecting
 3. Enable new assertion
 
 ### Can I copy assertions from other protocols?
-Yes, you can copy and adapt assertions from other protocols and use them in your own protocol. 
+Yes, you can copy and adapt assertions from other protocols and use them to protect your own protocol. 
 You should make sure that the assertions are compatible with your protocol. 
 It will make the most sense to copy assertions for standard libraries and interfaces.
 You will be able to see all assertions deployed by other protocols on the transparency dashboard.
 
 ## Implementation and Adoption
 
 ### How does the Credible Layer work at the sequencer level?
-The Credible Layer uses our modified Flashbots rBuilder that:
 1. Simulates each transaction against relevant assertions
 2. Allows valid transactions to proceed
 3. Filters out transactions that violate assertions
-4. Maintains Superchain compliance for L2 compatibility
 
 ### How does the Credible Layer work with decentralized sequencers?
 - **Enshrine assertions** as part of state transition function or fork-choice rule, making them integral to the protocol.
 - **Credible sequencers** that mark blocks as safe by running invariants and posting attestations on-chain, ensuring safety without compromising performance
 
 ### How are forced included transactions handled for OP Stack Rollups?
 
-The solution is currently in implementation. Ask us about it!
+The solution is currently in development. Ask us about it!
 
-### Does the Credible Layer introduce new trust assumptions?
+### Does the Credible Layer introduce new trust assumptions or centralization points?
 
-No. For single-sequencer L2s, users already trust the sequencer with transaction inclusion, because, in theory, a sequencer can censor transactions arbitrarily when it receives them at the RPC layer. 
+No. For single-sequencer L2s, users/dApps already trust the sequencer with transaction inclusion, because, in theory, a sequencer can censor transactions arbitrarily when it receives them at the RPC layer. 
```

Comment:
Just a side note: One could argue that the added sidecar code is an additional centralization point. No added trust assumptions, but it's an extra point in the system that's centralized.